### ISSUE TITLE
[FR]: An efficent way to query all db files. Probably a tag unique to the db-folder plugin in the yaml section.

### ISSUE THREAD

**Issue by rhhub** [RafaelGB/obsidian-db-folder#840]
### Present your request

I think a way to query based off a tag in the dbfolder file would be cool. We could have a dbfolder file that displays all the dbfolder files. The world will be just. Maybe a tag in the yaml section; sufficently unique to avoid conflicts.

### For which platform do you request this request??

Desktop

**Comment by onoter** [comment]
Each database file has a property `database-plugin`
I have a solution [here](https://github.com/RafaelGB/obsidian-db-folder/issues/558#issuecomment-1434782643)
Does it meet your requirement?

**Comment by rhhub** [comment]
Ya. Barring another plugin using the same key, it is sufficient.

**Comment by RafaelGB** [comment]
yeah, with that field will be enough. Maybe in the future we can change the file extension if it’s necessary
